recipes/default.rb

```diff
@@ -20,6 +20,14 @@
 # These two attributes should only be set when connecting directly to Chef Compliance, otherwise they should be nil
 token = node['audit']['token']
 server = node['audit']['server']
+interval_seconds = 0 # always run this by default, unless interval is defined
+unless node['audit']['interval'].nil? && !node['audit']['interval']['enabled']
```

Comment:
If I change line 24 to `if` then it becomes clearer and works as expected.

`if !node['audit']['interval'].nil? && node['audit']['interval']['enabled']`